src/main/kotlin/no/nav/tilleggsstonader/sak/interntVedtak/InterntVedtakService.kt

```diff
@@ -71,8 +85,26 @@ class InterntVedtakService(
         return InterntVedtak(
             behandling = mapBehandlingsinformasjon(behandling),
             søknad = mapSøknadsinformasjon(behandling),
-            målgrupper = mapVilkårperioder(vilkårperioder.målgrupper), // TODO læremidler
-            aktiviteter = mapVilkårperioder(vilkårperioder.aktiviteter), // TODO læremidler
+            målgrupper = mapVilkårperioder(vilkårperioder.målgrupper),
+            aktiviteter = mapVilkårperioder(vilkårperioder.aktiviteter),
+            stønadsperioder = mapStønadsperioder(behandling.id),
+            vilkår = mapVilkår(behandling.id, behandlingbarn),
+            vedtak = mapVedtak(vedtak),
+        )
+    }
+
+    fun lagInterntVedtakTilsynBarnV2(
+        behandling: Saksbehandling,
```

Comment:
Hvorfor trenger vi ulike metoder for ulike stønader her? Burde kunne bruke den samme for de ulike?

Reply:
Det interne vedtaket for barnetilsyn mapper ut barna i behandlingen, som ikke skal gjøres for læremidler, så på ett eller annet nivå må vel koden skille mellom de ulike stønadene?